samples/AspNetCore/ControllerSample/Readme.md

```diff
@@ -11,9 +11,9 @@ The application also registers for pub/sub with the `deposit` and `withdraw` top
 
  ## Running the Sample
 
- To run the sample locally run this comment in this directory:
+ To run the sample locally run this command in this project root directory:
  ```sh
- dapr run --app-id controller --app-port 5000 dotnet run
+ dapr run --app-id routing --app-port 5000 -- dotnet run
```

Comment:
Why has this been changed to routing?

Reply:
Because I want to keep it same with this line(https://github.com/heavenwing/dapr-dotnet-sdk/blob/grpcsample/samples/Client/DaprClient/Program.cs#L85). 
I have restored it to original word.